docs/CHANGELOG.asciidoc

```diff
@@ -26,6 +27,324 @@ This section summarizes the changes in each release.
+* Unmount map embeddable component on destroy {pull}45183[#45183]
+* Set mapbox Popup maxWidth {pull}45253[#45253]
+Monitoring::
+* Ensure we pass down all the parameters for fetching logs {pull}43869[#43869]
+* Issue 25323: Fixed error catch and route handling v2 {pull}44800[#44800]
+* Respect xpack.monitoring.show_license_expiration {pull}45537[#45537]
+Operations::
+* Configurable global socket timeouts {pull}31603[#31603]
+* Only provide obsoleteIndexTemplatePattern to the default index migrator to avoid race conditions {pull}42016[#42016]
+Platform::
+* Modify I18nProvider so that it does not generate new React components {pull}43556[#43556]
+* Use pipes for communication with chrome to avoid networking snafus {pull}42097[#42097]
+* Fix rawChecksums in chromium build and rename for clarity {pull}44665[#44665]
+* Fix duplicate fetch in Visualize {pull}41204[#41204]
+* Ensure visualizations in percentage mode respect the advanced settings percent format {pull}39044[#39044]
+* Resolve filter index references when importing saved objects {pull}42974[#42974]
+Queries::
+* Saved query requires title {pull}45029[#45029]
+* Save query form validation on blur {pull}43726[#43726]
+Reporting::
+* Fix Relative Date format in CSV export {pull}40544[#40544]
+* Fix TypeError with pending job info {pull}43924[#43924]
+SIEM::
+* Toggle Column / Code Coverage and Cypress {pull}42766[#42766]
+* Show all popover items / Timeline flyout button rendering {pull}45745[#45745]
+* Upgrade elastic charts library - v12.0.2 {pull}45853[#45853]
+* Adds performance enhancements such by removing wasted renderers and adding incremental DOM rendering {pull}43157[#43157]
+* Fixes the Suricata row renderer not being activated {pull}44728[#44728]
+* Changes ML conditional links to use tabs, fixes a small bug with null filterQuery {pull}45218[#45218]
+* Fixes index substring incorrectly matching configured indices and failing to install ML job {pull}43409[#43409]
+* Fix timeline/kql search disparity {pull}42843[#42843]
+* Formats the bytes columns in timeline {pull}43147[#43147]
+* Capitalizes SIEM in {kib} Advanced Settings {pull}44886[#44886]
+* Removes expensive global CSS calculations of resize handle {pull}43898[#43898]
+Security::
+* Spaces - allow for hex color codes that include uppercase characters {pull}43470[#43470]
+* Prevent space identifier url example from overflowing panel {pull}44871[#44871]
+Uptime::
+* Encode Monitor Id React Router params to validate {pull}43113[#43113]
+* Context Menu Improved for integrations links in monitors list {pull}43068[#43068]
+* Update monitor list configs for mobile view {pull}43218[#43218]
+Visualizations::
+* No longer registering Timelion feature if it's disabled {pull}42193[#42193]
+* Fix Timelion color error when filtering data {pull}23876[#23876]
+* Fix truncated formatting on tooltip {pull}41769[#41769]
+
+[float]
+[[enhancement-7.4.0]]
+=== Enhancements
```

Comment:
Move Enhancements before Bug Fixes to be consistent with previous release notes.